src/docs.md

```diff
@@ -953,7 +953,7 @@ Define multiple campaign uids to filter results by campaigns.
 Campaign slug to filter results by campaign. Requires `country`.
 
 `charityUid` (string)<br>
-Charity uid to filter results by charity.
+Charity uid to filter results by charity. If used with `campaignUid` or `campaignUids`, multiple charity uids can be passed as a CSV. This will return pages for these charities within the specified campaign(s).
```

Comment:
Nitpick but I think csv is a weird term here? I think of csv as a file type. Maybe just "comma separated list" or something – thoughts?